server/controllers/groups.js

```diff
@@ -279,7 +279,7 @@ export const create = (req, res, next) => {
       group,
       confirmation_url: user.generateLoginLink(`/${group.slug}`)
     }
-    emailLib.send('group.created', user.email, data);
+    emailLib.send('group.confirm', user.email, data);
```

Comment:
It would be better to move all email sending to notifications library. Non-blocking and easier to manage logic in one place. Let's try to work any new emails through that in the future. This one is fine for now.